.github/workflows/lint-and-test.yml

```diff
@@ -118,8 +118,8 @@ jobs:
         run: cargo test -p proof-of-sql --no-run --no-default-features --features="std"
       - name: Run cargo test (proof primitives - Dory) (std feature only - i.e. not using blitzar)
         run: |
-            cargo test proof_primitive::dory::dory_compute_commitments_test --no-default-features --features="std" && \
-            cargo test proof_primitive::dory::dynamic_dory_compute_commitments_test --no-default-features --features="std"
+            cargo test proof_primitive::dory::dory_compute_commitments_test --no-default-features --features="std,perf" && \
+            cargo test proof_primitive::dory::dynamic_dory_compute_commitments_test --no-default-features --features="std,perf"
```

Comment:
The `perf` enables blitzar. See here: https://docs.rs/crate/proof-of-sql/0.71.12/features. Or here: https://github.com/spaceandtimelabs/sxt-proof-of-sql/blob/84e34983dd58b867e41cfaff914e839108b6c42a/crates/proof-of-sql/Cargo.toml#L83-L90

Don't remove this or else CI won't test the non-blitzar path. If the non-blitzar path is failing, it means the non-blitzar code is broken.We use the non-blitzar path in production.